scripts/teahouse/send_th_invites.py

```diff
@@ -18,17 +18,22 @@
 import os
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
 
+import random
+import re
+
 import hb_toolkit
 import hb_output_settings
 import hb_profiles
-import random
+import page_reader
+
+RE_USER_NAME = re.compile(r"\[\[User:(.*?)(\]\]|\|)")
 
 def getEligibleInviters(elig_check, potential_inviters):
     """Filter out inviters with no edits in the last 21 days."""
     eligible_inviters = [x for x in potential_inviters if elig_check.determineInviterEligibility(x, 21)]
     return eligible_inviters
 
-def getEligibleInvitees(elig_check, potential_invitees, skip_templates):
+def getEligibleInvitees(elig_check, potential_invitees):
```

Comment:
Might be worth adding a comment on line 79 so that we remember that we are in fact still sending `skip templates` (and `output namespace`) to the underlying function.